blog_system/settings.py

```diff
@@ -0,0 +1,141 @@
+"""
+Django settings for blog_system project.
+
+Generated by "django-admin startproject" using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / "subdir".
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = ("django-insecure-sn-q1anqng&qknww)"
+              + "(_!+0j+h5isha+io&9&p*0p)-x&vi45gt")
```

Comment:
It's not a good practice to provide your secret key directly in your Django settings and store it in a version control repository, as this can lead to security vulnerabilities. Instead, you should use environment variables or a separate configuration file to manage your secret keys and sensitive information.